### PR TITLE
Block senders owa

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e176d59c0b/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b81cf9739b/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-63e1b484ff/how-to/).

**Comment by stephamon** [comment]
Hi @cyrichardson can you take a look at this one as well? Thanks!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4cbffc44fb/how-to/)

**Comment by cyrichardson** [comment]
@stephamon Looking now.

File: content/exchange/block-senders-in-owa.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+2. Click the gear icon in the upper right-hand corner, and then select **Options** from the drop down menu.
+
+    <img src="{% asset_path exchange/block-senders-in-owa/options_gear.png %}" />
+
+3. In the **Options** menu, click **Mail > Accounts > Block or Allow**.
+
+    <img src="{% asset_path exchange/block-senders-in-owa/block_or_allow.png %}" />
+
+4. Enter the email address or domain of the sender which you want to automatically mark as junk.
```
Comment (cyrichardson) [inline content/exchange/block-senders-in-owa.md]: change "which" to "that"

**Comment by cyrichardson** [comment]
@stephamon Just one small change. Everything else looks good!

File: content/exchange/block-senders-in-owa.md (hunk 1 above)
Comment (stephamon) [reply content/exchange/block-senders-in-owa.md]: Yeah, I caught that right after I tagged you! It's fixed :)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0e77bda93f/how-to/)

File: content/exchange/block-senders-in-owa.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- **Applies to:** User
+- **Difficulty:** Easy
+- **Time needed:** Approximately 15 minutes for change to complete
+- **Tools Required:** Mailbox access
+
+For more information about prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology/).
+
+### Block Senders
+
+**Warning:** Outlook Web App does not include a setting to reject senders. It only automatically marks blocked senders as junk. Administrators can reference [Blacklists in Microsoft Exchange](/how-to/spam-preferences-safe-lists-and-black-list-in-microsoft-exchange/#manage-black-list) for instructions on how to reject senders.
```
Comment (catlook) [inline content/exchange/block-senders-in-owa.md]: ... It automatically only ...

**Pull request merged**